azavea/iow-boundary-tool
# Wire up boundary details page

## Overview

This PR wires up the boundary details page. Some of the data on this page was not present in the data schema so that has been updated as well.

Closes #89 

### Notes

I not entirely sure what to do about the activity log. It needs more formatting/syntax to match what is in figma. I think this functionality should all be in one place (and not split between the front end and back end). 

## Testing Instructions

- Login as a contributor
- View some boundaries
  - [x] Ensure "Review" button does not appear for boundaries in review or approved
- Login as validator
- View some boundaries
  - [x] Ensure "Edit" button does not appear for drafts

## Checklist
